vmas/scenarios/navigation.py

```diff
@@ -215,6 +219,9 @@ def agent_reward(self, agent: Agent):
         return agent.pos_rew
 
     def observation(self, agent: Agent):
+        """
+        obs: [agent_pos, agent_vel, relative_to_goal_pos, lidar_readings]
+        """
```

Comment:
```suggestion
```

The observation is dependent on a few config variables, so if we go for a docstring we need one that fits all.
For now i sugget removing it since the function is quite simple to read.